pypicloud/access/ldap_.py

```diff
@@ -189,6 +206,8 @@ def group_members(self, group):
         return []  # pragma: no cover
 
     def is_admin(self, username):
+        if not username:
+            return False
```

Comment:
Why was this change needed? It's fine, I'm just curious what was causing is_admin to be called with a falsey username.